cmd/kubeadm/app/cmd/token.go

```diff
@@ -221,7 +221,19 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
-	// TODO: Validate usages here so we don't allow something unsupported
+	invalidUsages := sets.NewString()
+	usageAuthentication := strings.TrimPrefix(bootstrapapi.BootstrapTokenUsageAuthentication, bootstrapapi.BootstrapTokenUsagePrefix)
+	usageSigning := strings.TrimPrefix(bootstrapapi.BootstrapTokenUsageSigningKey, bootstrapapi.BootstrapTokenUsagePrefix)
+	// Validate usages
+	for _, usage := range usages {
+		if err := bootstrapapi.ValidateUsage(usage); err != nil {
```

Comment:
The error message of `ValidateUsage(usage)` is useless now, so I think you can change the func name to `IsValidateUsage` and let it return bool.

Reply:
Addressed